components/eamxx/docs/developer/dev_testing/ci_nightly.md

```diff
@@ -0,0 +1,18 @@
+# Continuous Integration and Nightly Testing
+
+## Autotester
+
+EAMxx using GitHub actions and a Sandia product called Autotester 2 (AT2)
+to run CI testing on a CPU and GPU machine for every GitHub pull
+request.
+By default, we run the `e3sm_eamxx_v1_at` suite and the
```

Comment:
That's not correct. We run a small subset of that. Currently, we run 5 ne4 cases, including one DPxx case, and one MMF2 case (which doesn't use eamxx, but does use p3/shoc from eamxx). 

You could just say 

```yaml
As of Feb 2025, our CI testing includes 5 CIME cases and the standalone eamxx tests (`test-all-scream`).
```